assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -1,15 +1,15 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 
 name: convert_model_to_mlflow
-version: 0.0.17
+version: 0.0.18
```

Comment:
not needed I guess? can you please compare with components in prod?

Reply:
Environment version upgrade would cause version upgrade, right?

Reply:
got it what you meant